### DISCUSSION TITLE
Postal (https://github.com/postalhq/postal) as Webhook provider

### DISCUSSION THREAD

**Discussion by redshiftltd** [mettle/sendportal#140] [Ideas] [answered]
Hi, 

I'm trying to roll my own email + newsletter service for a small organization.
Is it possible to configure sendportal to use webhooks provided by postal (https://github.com/postalhq/postal) ?

**Comment by JonoB** [comment, marked as answer]
Hi, its possible, but someone (you? :smile: ) would need to add the integration. 

You can refer to any of the other modules (postmark, sendgrid, mailgun, etc) to see how its done. 

https://github.com/mettle/sendportal-core/tree/master/src/Http/Controllers/Api/Webhooks
https://github.com/mettle/sendportal-core/tree/master/src/Events/Webhooks
https://github.com/mettle/sendportal-core/tree/master/src/Listeners/Webhooks

**Comment by redshiftltd** [comment]
That's fair, I'll take a look and see if I can come up with something. Thanks for the starting off point!
